tests/conftest.py

```diff
@@ -29,27 +40,105 @@ def salt_factories_config():  # pragma: no cover
     }
 
 
-@pytest.fixture(scope="package")
+@pytest.fixture(scope="module")
 def master_config():  # pragma: no cover
     """
     Salt master configuration overrides for integration tests.
     """
     return {}
 
 
-@pytest.fixture(scope="package")
+@pytest.fixture(scope="module")
 def master(salt_factories, master_config):  # pragma: no cover
     return salt_factories.salt_master_daemon(random_string("master-"), overrides=master_config)
 
 
-@pytest.fixture(scope="package")
-def minion_config():  # pragma: no cover
+@pytest.fixture(scope="module")
+def minion_config(kind_cluster):  # pragma: no cover
     """
     Salt minion configuration overrides for integration tests.
     """
-    return {}
+    return {
+        "kubernetes.kubeconfig": str(kind_cluster.kubeconfig_path),
+        "kubernetes.context": "kind-salt-test",
+        "providers": {"pkg": "kubernetes"},
```

Comment:
question: What's the purpose of declaring the `kubernetes` module as a provider for `pkg`? I don't think this makes sense, could be by accident.

Reply:
Yes, left over from some of my initial attempts at tests, removed.